### DISCUSSION TITLE
How come that many questions stay unanswered?

### DISCUSSION THREAD

**Discussion by PaulskPt** [micropython/micropython#16418] [General]
Hi, while browsing this follow-up of the archived [micropython forum](https://forum.micropython.org/), it appears to me that the majority of questions asked in these discussions have a status "unanswered". When I look at the archived forum, I see that the majority, if not all, questions are answered. What is happening?

**Comment by GitHubsSilverBullet** [comment]
It might have to do with the fact that a great many questions could have been avoided by just looking into the documentation. Or with a modicum of logical thinking.
This by the way is a phenomenon seen in other fora as well.
I'd think that if a question has even a tiny bit of 'weight', then it'll be answered.

**Reply by PaulskPt** [reply]
Thank you for your reply, however this has been also the case with the now archived forum.

**Reply by robert-hh** [reply]
Or the answer was not labelled as such.

**Reply by PaulskPt** [reply]
OK, thank you Robert.

**Reply by robert-hh** [reply]
If I scan though the discussion, most of them received a response. Like in the old forum which did not have the " answered" label.

**Reply by PaulskPt** [reply]
I created a discussion about a https 400 error a week ago, however no response sofar. I just changed the make-up of intro part. I hope this attracts more attention.
